initrd/init

```diff
@@ -69,35 +55,76 @@ if [ ! -z "$CONFIG_USB_BOOT_DEV" ]; then
 	echo >> /etc/fstab "$CONFIG_USB_BOOT_DEV /media auto defaults,ro 0 0"
 fi
 
-if [ ! -x "$CONFIG_BOOTSCRIPT" ]; then
-	recovery 'Boot script missing?  Entering recovery shell'
-	# just in case...
-	tpm extend -ix 4 -ic recovery
-	exec /bin/ash
+# Now it is safe to print a banner
+if [ ! -z "$CONFIG_LINUXBOOT" ]; then
```

Comment:
In my recent `export CONFIG_*` commit I didn't export the linuxboot config. Do you think it is important to be able to distinguish them?

Also, `[ ! -z "$CONFIG_LINUXBOOT" ]` will go down this path if `CONFIG_LINUXBOOT=n`.  It is better to test against `y`.